client/layout/masterbar/oauth-client.scss

```diff
@@ -127,251 +127,6 @@
 		--color-link-hover: #4ccee4;
 		clip-path: unset;
 	}
-
-	&.is-section-login {
-		padding-bottom: 100px;
-
-		@include breakpoint-deprecated(">660px") {
-			min-height: calc(100% - 145px);
-			padding-bottom: 145px;
-		}
-
-		.card {
-			&:not(.gdpr-banner) {
-				background: none;
-				box-shadow: none;
-				max-width: 320px;
-				padding: 0;
-			}
-
-			&:not(.two-factor-authentication__verification-code-form) {
-				margin: 0 auto;
-			}
-
-			// Allow for the illustration width
-			&.two-factor-authentication__push-notification-approval,
-			&.two-factor-authentication__actions {
-				max-width: 360px;
-			}
```

Comment:
I think this was for the 2FA app screen, would be worth testing that the illustration doesn't overflow the card. If this is a shared screen it might be alright though.

Reply:
<img src="https://github.com/user-attachments/assets/5dd5dbe3-b73d-45f3-88fc-12ee5c1fefbe" width="500">